package-lock.json

```diff
@@ -5,19 +5,27 @@
   "requires": true,
   "dependencies": {
     "@keymanapp/lexical-model-compiler": {
-      "version": "13.0.109",
-      "resolved": "https://registry.npmjs.org/@keymanapp/lexical-model-compiler/-/lexical-model-compiler-13.0.109.tgz",
-      "integrity": "sha512-QRH72i5RPZKU5Q210tAtpaucCYXU6wJQ9F+wR5T1OmkU/ICXwfqr5aElr7JoQMHJYHCYpXZF6y70eRKSKZDDdw==",
+      "version": "14.0.226-beta",
+      "resolved": "https://registry.npmjs.org/@keymanapp/lexical-model-compiler/-/lexical-model-compiler-14.0.226-beta.tgz",
+      "integrity": "sha512-x4NV0zoVwPSy/FUjUkiZ2O4CudJsZHq8feb4O3u2f9FUsoZlH1xA5s3u9KX+zHL9oPJKTfsrRNRvNgER/HckkA==",
       "requires": {
+        "@keymanapp/models-types": "^14.0.226-beta",
         "commander": "^3.0.0",
-        "typescript": "^3.2.4",
+        "typescript": "^3.8.3",
         "xml2js": "^0.4.19"
+      },
+      "dependencies": {
```

Comment:
Can't this block (ll. 17-22) be removed? The dependency is already at l. 25

Reply:
`package-lock.json` is automatically managed by NPM.  I'm not touching it.